data-prepper-plugins/otel-trace-source/build.gradle

```diff
@@ -18,8 +18,21 @@ dependencies {
     testImplementation "org.hamcrest:hamcrest:${versionMap.hamcrest}"
     testImplementation(platform('org.junit:junit-bom:5.7.0'))
     testImplementation('org.junit.jupiter:junit-jupiter')
+    testImplementation("org.mockito:mockito-core:3.7.0")
```

Comment:
Nit: do u think we need to move the versions to versionMap to be reused

Reply:
@yadavcbala Thanks for the reminder. There is already a mockito in versionMap. But for Junit 5 (jupiter.api), I do need a higher version of mockito. Will make a comment here to see if it can be applied to all other subprojects.